src/sentry/monitors/utils.py

```diff
@@ -65,7 +65,7 @@ def valid_duration(duration: Optional[int]) -> bool:
 
 def fetch_associated_groups(
     trace_ids: List[str], organization_id: int, project_id: int, start: datetime, end
-) -> Dict[str, Set[int]]:
+) -> Dict[str, List[Dict[str, int]]]:
```

Comment:
not that familiar with python types, but should this be like:
 `List[Dict[str, Union[int, str]]]`? since shortId is a string?

Reply:
ah. yes.

Reply:
spun myself around because `short_id` is an `int` but `qualified_short_id` is a `str`